### ISSUE TITLE
Want values of trigonometric functions as degrees, and convert them too

### ISSUE THREAD

**Issue by mouri11** [BytesClub/Tabcalc#2]
I haven't found any solution to this. sin 30 gives -0.98... instead of 0.5. Need to be able to change among these values.

**Comment by devutsav** [comment]
Maybe, keep a toggle button for toggling between radians and degree.

**Comment by mouri11** [comment]
Yes, I am keeping toggle buttons but I am not sure how to use it after sin x is calculated.

**Comment by mouri11** [comment]
Like, in Casio fx-82ms,if you calculate sin x, you can specify whether x is in degrees or radians. Any way to specify that?

**Comment by devutsav** [comment]
In casio  fx-82ms, sin 30 is calculated, then if we change mode from radian to degrees(say), now we need to press on "equals to" again to calculate the new value. You can do something like that.
